ci_framework/hooks/playbooks/ceph-deploy.yml

```diff
@@ -0,0 +1,23 @@
+# Following those notes: https://github.com/fultonj/zed/tree/main/edpm
+- hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Load all of our cifmw variables
+      ansible.builtin.include_vars:
+        file: "{{ cifmw_basedir }}/artifacts/custom-params.yml"
+    - name: Run make_ceph
+      vars:
+        make_ceph_environment:
+          TIMEOUT: 90
+      ansible.builtin.include_role:
+        role: install_yamls_makes
```

Comment:
out of my curiosity I was looking for this role (in particular the "make_ceph.yaml" that is used to run the `make ceph TIMEOUT=90` (and other options). Do you have a pointer to that role?

Reply:
I see that this file is basically dynamically generated for each component you want to deploy and the resulting `make` is called w/ the related arguments

Reply:
Exactly :). That wrapper is really neat.